src/content/plugins/chunks-webpack-plugin.md

```diff
@@ -0,0 +1,322 @@
+---
+title: ChunksWebpackPlugin
+contributors:
+  - yoriiis
+related:
+  - title: SplitChunksPlugin
+    url: /plugins/split-chunks-plugin
+  - title: The real power of Webpack 4 SplitChunks Plugin
+    url: https://medium.com/@Yoriiis/the-real-power-of-webpack-4-splitchunks-plugin-fad097c45ba0
+---
+
+The ChunksWebpackPlugin creates HTML files with entry points and chunks relations to serve your webpack bundles. It is suitable with multiple page application which contains multiple entry points.
+
+Since Webpack 4, SplitChunksPlugin offer the possibility to optimizes all chunks. It can be particularly powerful, because it means that chunks can be shared even between async and non-async chunks. See the webpack documentation for [`splitChunks.chunks`](https://webpack.js.org/plugins/split-chunks-plugin/#splitchunkschunks).
+
+This option automatically generate new chunks associated with an entry point. For example, entry points `a.js` and `b.js` share common codes with the file `vendors~a~b.js`.
+
+With multiple entry points, it can be difficult to identify relation between auto-generated chunks and entry points.
+
+The plugin parse the `entrypoints` [Map](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Map) object from the webpack compilation to get all valid entry points and associated files. Then, it generates HTML files which include all assets filtered by entry point and the `chunks-manifest.json` file.
+
+## Zero config
+
+The plugin works without configuration. For advanced usage, see the section [using configuration](#using-a-configuration).
+
+## Installation
+
+The plugin is available as a package with the name of `chunks-webpack-plugin` on [npm](https://www.npmjs.com/package/chunks-webpack-plugin) and [Github](https://github.com/yoriiis/chunks-webpack-plugin).
```

Comment:
links should be meaningful for seo purposes
```suggestion
`ChunksWebpackPlugin` is available on npm as [`chunks-webpack-plugin`](https://www.npmjs.com/package/chunks-webpack-plugin) and as [`chunks-webpack-plugin` on Github](https://github.com/yoriiis/chunks-webpack-plugin).
```

Reply:
I think there is a mistake with the bracket on the Github link, I will updates the line in a separate commit

Reply:
Okay I understand, it is perhaps not a mistake. I find the notation with the double links (npm and npm) a little ambiguous. I have test other example, what do you think?

1-
```text
`ChunksWebpackPlugin` is available on npm as [`chunks-webpack-plugin`](https://www.npmjs.com/package/chunks-webpack-plugin). See the [Github](https://github.com/yoriiis/chunks-webpack-plugin) page for details.
```

2-
```text
`ChunksWebpackPlugin` is available on [npm](https://www.npmjs.com/package/chunks-webpack-plugin) and [Github](https://github.com/yoriiis/chunks-webpack-plugin) as `chunks-webpack-plugin`.
```

3- Only one link perhaps, Github or npm ?

Reply:
keep both, but make sure github link doesnt just say github, e.g. output `<a href="https://github.com/yoriiis/chunks-webpack-plugin">Github</a>` is not seo friendly, say exactly what its linking, it does link `chunks-webpack-plugin on Github`